lib/kdump_utils.pm

```diff
@@ -164,4 +170,130 @@ sub do_kdump {
     script_run "echo c > /proc/sysrq-trigger", 0;
 }
 
+sub install_service {
+    zypper_call("in kdump");
```

Comment:
looks like duplication(again)  of `prepare_for_kdump` function

Reply:
OK, we'll use prepare_for_kdump.